arangosh/Restore/RestoreFeature.cpp

```diff
@@ -308,9 +308,10 @@ arangodb::Result tryCreateDatabase(arangodb::application_features::ApplicationSe
     if (properties.isObject()) {
       ObjectBuilder guard(&builder, "options");
       for(auto const& key : std::vector<std::string>{
-        arangodb::StaticStrings::MinReplicationFactor,
+        arangodb::StaticStrings::MinReplicationFactor, // deprecated
```

Comment:
I think we can remove `minReplicationFactor` here, as it is a new API in 3.6. And 3.5 never used `minReplicationFactor` for databases.